src/components/QuickOpenModal.js

```diff
@@ -69,12 +70,16 @@ type GotoLocationType = {
   column?: number
 };
 
+const updateResultsDebounce = 100;
 const maxResults = 100;
 
 function filter(values, query) {
+  const preparedQuery = fuzzyAldrin.prepareQuery(query);
```

Comment:
[Parses the query in advance](https://github.com/jeancroy/fuzz-aldrin-plus/blob/master/src/query.coffee#L13) - for some reason `filter` doesn't seem to perform this optimization (although it could).